### DISCUSSION TITLE
Log line length issue

### DISCUSSION THREAD

**Discussion by moiz2050** [opcodesio/log-viewer#281] [Q&A] [answered]
I am having an issue with loading logs for my custom log type, It loads the folders but dont load any logs for the file, when I dig into the code I figured out that in `IndexedLogReader` at line 92 has this `fgets($this->fileHandle, 1024)`, due to fixed length it's splitting my Line into multiple, that result in not matching the regex. if I remove the `1024` length from `fget` its working fine, is there a good way to tackle this?

**Comment by arukompas** [comment, marked as answer]
hey @moiz2050 , it should not have been there. Must be a leftover from v1.

I've pushed an update, version **3.1.4**. Please update and let me know if that fixes the issue! :)

**Reply by moiz2050** [reply]
Thank you for the response @arukompas its fixed now
